src/languages/scala.js

```diff
@@ -117,6 +117,11 @@ export default function(hljs) {
     contains: [ NAME ]
   };
 
+  const INLINE = {
+    className: 'keyword',
+    begin: /(?<!\.)\binline(?=\s)/
```

Comment:
Again, can't use look-behind.  To solve "false positive" cases like this you can often introduce a second dummy rule like:

```js
// consume inline as a property access
{ match: /\.inline/ } 
```

The dummy rule matches the proper access case, preventing your highlighting rule from seeing that content.

---

Also, `beginKeywords` has this magic built-in but you'd need to be inside of a separate scope to take advantage of that, so I think my first suggestion may be what you want.

Reply:
`beginKeywords` did not work with the `` `inline` + ...`` case.